src/shared/utils.ts

```diff
@@ -500,15 +505,18 @@ export function toast(text: string, background = "success") {
   }
 }
 
-export function pictrsDeleteToast(
-  clickToDeleteText: string,
-  deletePictureText: string,
-  failedDeletePictureText: string,
-  deleteUrl: string
-) {
+export function pictrsDeleteToast(filename: string, deleteUrl: string) {
```

Comment:
The 3 string arguments were the same everywhere this function was used, and there didn't seem to be a reason for non picture deletion related strings to need to be used here, so I define the strings in the function and only take the filename as an argument.